tools/test.py

```diff
@@ -97,9 +103,11 @@ def main():
     # prepare the dataset loader
     dataset_type = 'test'
     dataset = task_processor.build_dataset(model_cfg, dataset_type)
+    # override samples_per_gpu that used for training
+    model_cfg.data['samples_per_gpu'] = args.batch_size
     data_loader = task_processor.build_dataloader(
         dataset,
-        samples_per_gpu=1,
+        samples_per_gpu=model_cfg.data.samples_per_gpu,
```

Comment:
Shall we update the latency performance if we use batch_size >1?

Reply:
Maybe not for now. not sure if all models support exporting with batch_size >1.

Reply:
How about adding a TODO here? Besides, if not for all the models, shall we add limitations to the use of `--batch-size`?

Reply:
Adding a note in the description is Ok.